src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -1302,9 +1315,7 @@ public void setDisplayName(@Nullable JsonMessage displayName) {
     public void setDisplayName(@Nullable Component displayName) {
         this.displayName = displayName;
 
-        PlayerInfoPacket infoPacket = new PlayerInfoPacket(PlayerInfoPacket.Action.UPDATE_DISPLAY_NAME);
-        infoPacket.playerInfos.add(new PlayerInfoPacket.UpdateDisplayName(getUuid(), displayName));
-        sendPacketToViewersAndSelf(infoPacket);
+        this.getTabList().updateDisplayName(this);
```

Comment:
Do these methods (display name, latency, gamemode, etc) not need to update the display name for any tab list the player is in?